### ISSUE TITLE
Adding a Service List without a language crashes the backend

### ISSUE THREAD

**Issue by peterl1000** [DVBProject/DVB-I-Reference-CSR#1]
Steps to recreate:
- Select "Add Service List".
- Fill in "Service List Name", "URI", "Organization" and "Deliveries".
- Do *not* fill in Language.
- Click "Add List".
- Backend crashes.

ServiceListOfferingType is allowed to have zero Language elements (DVB-I spec section 5.3.5).

Error is:
```
TypeError: Cannot read property 'a3' of undefined
    at Query.<anonymous> (/home/peterl/gitrep/DVB-I-Reference-CSR/backend/app/models/servicelist.model.js:30:81)
    at Query.<anonymous> (/home/peterl/gitrep/DVB-I-Reference-CSR/backend/node_modules/mysql/lib/Connection.js:526:10)
    at Query._callback (/home/peterl/gitrep/DVB-I-Reference-CSR/backend/node_modules/mysql/lib/Connection.js:488:16)
    at Query.Sequence.end (/home/peterl/gitrep/DVB-I-Reference-CSR/backend/node_modules/mysql/lib/protocol/sequences/Sequence.js:83:24)
    at Query._handleFinalResultPacket (/home/peterl/gitrep/DVB-I-Reference-CSR/backend/node_modules/mysql/lib/protocol/sequences/Query.js:149:8)
    at Query.OkPacket (/home/peterl/gitrep/DVB-I-Reference-CSR/backend/node_modules/mysql/lib/protocol/sequences/Query.js:74:10)
    at Protocol._parsePacket (/home/peterl/gitrep/DVB-I-Reference-CSR/backend/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/home/peterl/gitrep/DVB-I-Reference-CSR/backend/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/home/peterl/gitrep/DVB-I-Reference-CSR/backend/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/home/peterl/gitrep/DVB-I-Reference-CSR/backend/node_modules/mysql/lib/protocol/Protocol.js:38:16)
```

**Comment by juhajoki** [comment]
thank you, to be fixed in next release.

**Comment by juhajoki** [comment]
fixed in version 0.17
